packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -130,15 +143,25 @@ const Conversation = WebexPlugin.extend({
       });
   },
 
+  /**
+   * delete content
+   * @param {Object} conversation
+   * @param {Object} object
```

Comment:
I still don't know what this `object` object is.
![img](https://media2.giphy.com/media/h4Z6RfuQycdiM/giphy.gif)